kani-driver/src/harness_runner.rs

```diff
@@ -39,15 +45,19 @@ impl<'sess, 'pr> HarnessRunner<'sess, 'pr> {
         self.check_stubbing(harnesses)?;
 
         let sorted_harnesses = crate::metadata::sort_harnesses_by_loc(harnesses);
-
+        let max_threads = 8;
```

Comment:
Why did you have to change this?

Reply:
Ya this was a test of something else, accidentally slipped through, idk how, sorry. Will fix it up.